core/libraries/plugin_api/EE_Register_Addon.lib.php

```diff
@@ -326,7 +345,7 @@ private static function _verify_parameters($addon_name, array $setup_args)
             );
         }
         // check that addon has not already been registered with that name
-        if (isset(self::$_settings[ $addon_name ]) && ! did_action('activate_plugin')) {
+        if (isset(self::$_settings[$addon_name]) && ! did_action('activate_plugin')) {
```

Comment:
the existing code was correct as we are adding spaces around variables used as keys

Reply:
huh pretty sure I didn't manually remove those spaces, but maybe I did